gxa/src/main/java/uk/ac/ebi/atlas/home/HomeController.java

```diff
@@ -76,17 +104,34 @@ public String getHome(Model model) {
 
         model.addAllAttributes(latestExperimentsService.fetchLatestExperimentsAttributes());
 
-        model.addAttribute(
-                "speciesList", GSON.toJson(popularSpeciesService.getPopularSpecies(FEATURED_SPECIES)));
-        model.addAttribute(
-                "animalsList", GSON.toJson(popularSpeciesService.getPopularSpecies("animals", FEATURED_SPECIES)));
-        model.addAttribute(
-                "plantsList", GSON.toJson(popularSpeciesService.getPopularSpecies("plants", FEATURED_SPECIES)));
-        model.addAttribute(
-                "fungiList", GSON.toJson(popularSpeciesService.getPopularSpecies("fungi", FEATURED_SPECIES)));
-
         model.addAttribute("speciesPath", ""); // Required by Spring form tag
 
+        List<ExperimentInfo> experimentlist = experimentInfoListService.listPublicExperiments();
+
+        Double numberOfAssays = 0.0;
```

Comment:
Why is `numberOfAssays` a `Double`, which you then round later on?

Reply:
The data I retrieved from experimentInfoListSevice is `Double`. I want to get an integer then.

Reply:
I can't see any `Double`s in the `ExperimentInfo` class. 

![screenshot 2019-01-31 at 13 01 51](https://user-images.githubusercontent.com/5923538/52055843-731ad980-2558-11e9-9602-06301b1e42c7.png)